tests/umock_c_int/umock_c_int.c

```diff
@@ -106,7 +106,7 @@ int are_equal_func_TEST_STRUCT_COPY_FAILS(const TEST_STRUCT_COPY_FAILS* left, co
 {
     (void)left;
     (void)right;
```

Comment:
looks like the (void)can go away

Reply:
And in the below change

---
In reply to: [399327435](https://github.com/Azure/umock-c/pull/154#discussion_r399327435) [](ancestors = 399327435)

Reply:
Removed.

---
In reply to: [399327569](https://github.com/Azure/umock-c/pull/154#discussion_r399327569) [](ancestors = 399327569,399327435)